src/django/api/processing.py

```diff
@@ -644,15 +644,17 @@ def handle_external_match_process_result(id, result, request, should_create):
     if queryset_f_m.count() == 0:
         # No Match and Geocoder Returned No Results
         return get_error_match_result(f_l_item.id, result)
-    if (queryset_f_m.count() == 1
-            and queryset_f_m[0].status == FacilityMatchTemp.AUTOMATIC):
+
+    if queryset_f_m[0].status == FacilityMatchTemp.AUTOMATIC:
```

Comment:
frankly saying I completely don't understand  changes:
1. queryset_f_m[0] it will raise exception when there's no items in array
2. it seems you simply change order, instead: 
```
if (queryset_f_m.count() == 1
             and queryset_f_m[0].status == FacilityMatchTemp.AUTOMATIC):
```
you make:
```
if (queryset_f_m[0].status == FacilityMatchTemp.AUTOMATIC
and queryset_f_m.count() == 1):
```